packages/server/src/councils/entities/council.entity.ts

```diff
@@ -0,0 +1,17 @@
+import { Column, Entity, Index, PrimaryGeneratedColumn } from 'typeorm';
+
+@Index('IDX_councils_name', ['name'])
+@Entity('councils')
+export class Council {
+  @PrimaryGeneratedColumn({ name: 'id', type: 'int' })
+  id: number;
+
+  @Column({ name: 'name', type: 'varchar' })
+  name: string;
+
+  @Column({ name: 'lat', type: 'double precision' })
+  lat: number;
+
+  @Column({ name: 'lon', type: 'double precision' })
+  lon: number;
+}
```

Comment:
1. Since we're not really leveraging TypeORM's migration functionality and are creating our own migration scripts, I don't think the `@Index` decorator does anything. Same story with the explicit types in the `@Column` decorators. Best to remove them.
2. We should add an `address` `TEXT` column here too so that we can easily display it for the user when looking at events.

Reply:
We’re not leveraging its migration auto generation functionality, but we are using its migration interface that generates raw SQL from TypeScript. I’ve never liked using the more magic features of ORM’s and prefer to retain control of the more critical components like database design/migration.